pilot/cmd/pilot-agent/main.go

```diff
@@ -182,24 +182,25 @@ var (
 				log.Infof("Effective config: %s", out)
 			}
 
-			certs := make([]envoy.CertSource, 0, 3)
+			internalCerts := make([]envoy.CertSource, 0, 3)
+			externalCerts := make([]envoy.CertSource, 0, 3)
```

Comment:
What are internal certs vs external certs?

Reply:
I assume the ingress certs - the question is why do we need to monitor them ? They expire in 1 year typically.

Reply:
Yes, the external certs mean the ingress certs for external traffic, and the internal certs mean the certs issued by Istio CA, used within the cluster
 I don't have preference on whether monitoring the ingress certs. The operators can manually restart the ingress controller if the ingress certs are updated, which may be once a year. With that said, if monitoring the ingress certs is no extra work for us, I think we can just support it.